src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
-function calculateRentalCost(days) {
-  // write code here
+const DAILY_RATE = 40;
+const LONG_TERM_DAYS = 7;
+const LONG_TERM_DISCOUNT = 50;
+const MEDIUM_TERM_DAYS = 3;
+const MEDIUM_TERM_DISCOUNT = 20;
+
+function calculateRentalCost(numberOfDays) {
+  const basePrice = numberOfDays * DAILY_RATE;
+
+  if (numberOfDays >= LONG_TERM_DAYS) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  }
+
+  if (numberOfDays >= MEDIUM_TERM_DAYS) {
+    return basePrice - MEDIUM_TERM_DISCOUNT;
+  }
```

Comment:
The logic here applies only one discount based on the number of days. If `numberOfDays` is greater than or equal to `LONG_TERM_DAYS`, it correctly applies the long-term discount. However, if `numberOfDays` is between `MEDIUM_TERM_DAYS` and `LONG_TERM_DAYS - 1`, it applies the medium-term discount. This logic is correct, but ensure that the requirements specify that only one discount should be applied. If both discounts should be considered, the logic needs to be adjusted.